Makefile

```diff
@@ -36,3 +36,9 @@ clean_all_service:
 
 clean_all_tools:
 	cd tools && make clean
+
+add_service:
```

Comment:
A lot of the make file is going to be taken out if we go through with my proposal. Most of it is irrelevant as Lucida stand-alone will not be making/building the black boxes. The black boxes build themselves. Perhaps this is something we need to discuss.